imbi/endpoints/project_ssm_configuration.py

```diff
@@ -134,6 +134,58 @@ async def get(self, *args, **kwargs) -> None:
         } for name, d in params_by_path.items()]
         self.send_response(output)
 
+    async def post(self, *args, **kwargs) -> None:
+        if not self.settings['google']['enabled']:
+            raise errors.Forbidden('Google integration is disabled',
+                                   title='Google integration is disabled')
+
+        if not self.settings['google']['integration_name']:
+            raise errors.Forbidden(
+                'No Google integration specified in configuration',
+                title='No Google integration specified in configuration')
+
+        tokens = await self.current_user.fetch_integration_tokens(
+            self.settings['google']['integration_name'])
+        if len(tokens) == 0:
+            raise errors.Forbidden(
+                'No OAuth 2.0 tokens for %s',
+                self.current_user.username,
+                title=('Not authorized to access SSM in this namespace &'
+                       ' environment'))
+        google_tokens = tokens[0]
```

Comment:
You can push this into a helper since it is also used in `get`

Reply:
good idea